pyfstat/make_sfts.py

```diff
@@ -191,6 +191,8 @@ class Writer(BaseSearchClass):
     for more detailed help with some of the parameters.
     """
 
+    mfd = "lalapps_Makefakedata_v5"
```

Comment:
```suggestion
    mfd = "lalapps_Makefakedata_v5"
    """The executable; can be overridden by child classes."""
```
(I was just checking the docs built from this branch and this shows up as a class attribute, so better have a short string on it.)